.github/workflows/release-cfe-cherry-pick.yml

```diff
@@ -222,24 +218,20 @@ jobs:
               with:
                   slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
                   slack-channel: ${{ secrets.WOO_RELEASE_SLACK_NOTIFICATION_CHANNEL }}
+                  slack-optional-unfurl_links: false
+                  slack-optional-unfurl_media: false
                   slack-text: |
-                      :warning-8c: CFE cherry pick failed for '${{ needs.prep.outputs.release }}'
-
-                      An attempt to cherry pick PR(s) into outgoing trunk for `${{ needs.prep.outputs.release }}` has failed. This could be due to a merge conflict or something else that requires manual attention. Please check: https://github.com/woocommerce/woocommerce/pull/${{ needs.prep.outputs.pr }}
-
+                      :warning: *Cherry pick <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|failed> (${{ needs.prep.outputs.version }} → `trunk`)* Please resolve before releasing. Source PR: _<${{ needs.prep.outputs.prURL }}|#${{ needs.prep.outputs.pr }} - ${{ needs.prep.outputs.prTitle }}>_.
```

Comment:
To make it potentially more actionable, I think we could add here clues about the actor/author - I guess resolving will be delegated, and we can potentially skip the digging step for ourselves. My guess can be inaccurate though.